### ISSUE TITLE
FSharp.Core XML help is not localized

### ISSUE THREAD

**Issue by masq6r** [dotnet/fsharp#2360]
The quick info shows only the signature but not the help information for some F# functions:
![1](https://cloud.githubusercontent.com/assets/594939/22578467/f5de380e-ea03-11e6-8c4d-48e12860bdeb.png)
Compared to a .NET method:
![2](https://cloud.githubusercontent.com/assets/594939/22578482/14529988-ea04-11e6-8f73-c10599804432.png)

If I open the same project in VS 2015 Community the quick info shows fully.

OS: Windows 10 with latest patch
VS 2017 Community RC3
Visual F# Tools extension

**Comment by vasily-kirichenko** [comment]
Those are two different features: quick info and signature help, so the difference. And regression.

**Comment by cartermp** [comment]
On the default template with RC3 + VSIX this doesn't fail:

![works](https://cloud.githubusercontent.com/assets/6309070/22580449/65851b72-e98d-11e6-89c6-c59bb7771d69.PNG)

Could you share the code where you noticed this?

**Comment by CodEFouR** [comment]
I investigated the issue a bit and found it could be a localization issue for F#. If change the language to English, I could see the full help. But when I switch to a local language like Chinese the help doesn't show for some functions. 
Some local text might be missing so the full help doesn't show.

**Comment by cartermp** [comment]
Ah!  That is a bug indeed, then.  Do you still get localized error messages and things like that, or is this only something you've seen in QuickInfo when highlighting an item in a completion list?

**Comment by CodEFouR** [comment]
Yes I can see the localized error messages... I guess some functions don't have the translated complete local help text (e.g. ```List.item```)

**Comment by cartermp** [comment]
Okay, in that case I'll update this issue and re-open.  Definitely a bug, but it's good that you're still getting other things localized.

**Comment by dsyme** [comment]
Renaming this to cover the general fact that FSHarp.Core XML doc help has never been localised

**Comment by dsyme** [comment]
Note  I believe we would need to:

* implement this tag https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#include

* get the XML doc comments out of the FSharp.Core source files so they can go through localization

**Comment by KevinRansom** [comment]
@dsyme, we would have to generate localizable xml files as well as improve probing, probably leveraging the .xlf infrastructure.  We could do it, and the localizers would pick it up and produce new text, it would actually be great to do.
